magento/pwa-studio
# Cypress multiple wishlist

## Description

Cypress test for multiple wishlist

## Related Issue







Closes [PWA-1808](https://jira.corp.magento.com/browse/PWA-1808)

## Acceptance





### Verification Stakeholders



### Specification




### Verification Steps




1. Clone this branch and `cd venia-integration-tests/`
2. `yarn install`
3. `yarn test`
4. From cypress UI run both wishlist E2E and integration tests and make sure they pass.

## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
